samples/cellular/modem_shell/overlay-nrf700x-wifi-scan-only.conf

```diff
@@ -13,7 +13,6 @@ CONFIG_LOCATION_METHOD_WIFI=y
 # Align this with CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT.
 # Also see comments for CONFIG_HEAP_MEM_POOL_SIZE.
 CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT=50
```

Comment:
I'm thinking if we should drop this to 30 or even 20. Each AP is 48 bytes of RAM in Location library.
We could even set `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT` to unlimited (zero) in MOSH as that doesn't affect on the RAM as nrf70 driver allocates the results from kernel heap.
This would mean that `wifi scan` command would return maximum number of APs and Location library would use, e.g., 30, which is still too much. We can keep the number a little high in a test application and real apps like ATv2 define a lower value.

In ATv2, location and nrf_cloud_multi_service samples we should still define `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT` because there is no Wi-Fi shell so the functionality is more optimized.

Opinions? @jhirsi, @tokangas

Reply:
I'm fine with setting `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT` to zero and `20` should be plenty for `CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT`, even less would probably give the same positioning accuracy. Maybe add comments for both options, though.

Reply:
The downside of `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT=0` is that we get these logs x-20 times where x is number of found APs:
```
[00:03:54.922,790] <wrn> location: Scanning result (mac 6c:8b:d3:38:34:af) did not fit to result buffer - dropping it
[00:03:54.934,020] <wrn> location: Scanning result (mac 6c:8b:d3:38:34:ae) did not fit to result buffer - dropping it
```
I'm leaning towards setting `CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT=20` also to avoid people wondering about these.

Reply:
Or we would need to change the log from warning to info or debug...

Reply:
I've set both `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT` and `CONFIG_LOCATION_METHOD_WIFI_SCANNING_RESULTS_MAX_CNT` to 20 at this point. It's another topic and PR if we want to make `CONFIG_NRF_WIFI_SCAN_MAX_BSS_CNT=0`.